Logger/Logger.cpp

```diff
@@ -59,6 +59,15 @@ bool Logger::LogError(string sError, signed int& iTurnCount)
     return RetVal;
 }
 
+bool Logger::LogError(string sError)
+{
+    bool RetVal = true;
```

Comment:
If you are always returning true. Why do you need the bool?
Also generally log method have void return types.

Reply:
Yes, that makes sense, I don't need bool.